src/data/Philippines-ministry-of-energy_metadata.json

```
@@ -0,0 +1,128 @@
+[
+  {
+    "id": "PMoE-Reference-2023",
+    "publisher": "Philippines - Ministry of Energy",
+    "name": "Reference",
+    "description": "Reference Scenario provide a scenario where current energy policies are maintained in Philippines",
```

Wording is a bit odd here, FYI @Antoine-Lalechere and @jacobvjk 

```suggestion
    "description": "A scenario where current energy policies are maintained in the Philippines",
```